ui-src/src/app/test-cases/test-cases.component.spec.ts

```diff
@@ -0,0 +1,26 @@
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+
+import { TestCasesComponent } from "./test-cases.component";
+
+describe("TestCasesComponent", () => {
+  let component: TestCasesComponent;
+  let fixture: ComponentFixture<TestCasesComponent>;
+
+  beforeEach(
+    async(() => {
```

Comment:
@ColmCooney  & @angerooney11  Have we checked that the unit tests run. Cause async afaik is not available until node 7